services/brig/src/Brig/API/Public.hs

```diff
@@ -665,6 +665,7 @@ sitemap o = do
       \number is not whitelisted, a 403 error is returned."
     Doc.body (Doc.ref Doc.newUser) $
       Doc.description "JSON body"
+    -- FUTUREWORK: I think this should be 'Doc.self' instead of 'user'
```

Comment:
If so, I think that's a bug and should not be forgotten in a `FUTUREWORK`.

Reply:
We have a lot of Swagger that's not up to date (or not served in swagger-ui at all). This one is relatively harmless: it just doesn't tell you about some fields it will return (`user` is a subset of `self`).

The real solution here is to make the Swagger generation less manual, e.g. with Servant.